installers/common/Types.hs

```diff
@@ -57,6 +58,11 @@ data Cluster
   | Testnet
   deriving (Bounded, Enum, Eq, Read, Show)
 
+data Backend
+  = Cardano { cardanoDaedalusBridge :: FilePath }
```

Comment:
I realize this was just moved from one place to another, but it's an anti-pattern to mix sum and record types like this. It results in partial functions. For instance, given a type of `Backend` you can now call `cardanoDaedalusBridge`, even when the value is `Mantis`.

One solution is to make a separate type for the record and use it as a parameter of the `Cardano` constructor.

Reply:
Given the simplicity of the constructor here and the fact that `cardanoDaedalusBridge` isn't used anywhere I recommend simply changing this to `Cardano FilePath`.

Reply:
Yeah I wrote the original, fully aware of the anti-pattern. Forgive my sins. I used the record type just to label what the parameter is, not to actually be called. Pushed a fix.